### ISSUE TITLE
Rank Skipping

### ISSUE THREAD

**Issue by happyzachariah** [nishi7409/ClanLabs-OpenSourced#6]
I've encountered an issue attempting to skip ranks. My group I'd like to utilize this for has 3 ranks in a row dedicated to sub-groups. These ranks are in place to separate those in sub-groups from those only participating in the main group.

So, when you go from the lower trainee ranks to the higher trained ranks, you have to skip over these. I tried leaving out the "division ranks" in the main group from XP requirements, but it's defaulted to 0 and promotes you to them anyways.

I've tried locking them, which worked, but then when a user was promoted from the one just before they were still promoted.

I also tried replicating promotion to three times if their rank = the lower rank to skip them over, but to no avail.

If there's any chance you could assist me or let me know if this is possible, it'd be greatly appreciated. 

tl;dr - Trying to skip over ranks in the main group that don't apply to those that continue to apply in a linear system. Haven't found a way to skip ranks.

(If this is confusing, I can clarify)

**Comment by nishi7409** [comment]
Add me on Discord, let's chat there.

nishi#1028
